### ISSUE TITLE
Dioxus ignores the autofocus attribute for <input> elements

### ISSUE THREAD

**Issue by BugsSoren** [DioxusLabs/dioxus#725]
**Problem**
Dioxus ignores the HTML autofocus attribute.

**Steps To Reproduce**
Use this code snippet

    cx.render(rsx! { article {
        h1 { "Some title" },
        input {
            r#type: "text",
            autofocus: "true",
        }
    }})

**Expected behavior**
The <input> element should receive focus on rendering.
I copied the HTML generated by Dioxus using browser devtools into a separate HTML file. Loading the file into the browser, yields the expected focus to the <input> element.

When I discovered this a few weeks ago, I verified that web-sys does handle autofocus properly.  I could use_future() kludge solved it there.

**Screenshots**

**Environment:**
 - Dioxus version: 0.2.4
 - Rust version:  nightly
 - OS info: Linux and Firefox
 - App platform: web

**Questionnaire**

- [ ] I'm interested in fixing this myself but don't know where to start
- [ ] I would like to fix and I have a solution
- [ x] I don't have time to fix this right now, but maybe later

**Comment by Demonthos** [comment]
This is fixed on the `0.3` release with a boolean attribute instead of text:

```rust
// [dependencies]
// dioxus = "0.3"
// dioxus-web = "0.3"
cx.render(rsx! { article {
    h1 { "Some title" },
    input {
        r#type: "text",
        autofocus: true,
    }
}})
```

**Comment by BugsSoren** [comment]
FYI
I migrated my project to dioxus-web v 0.3.1,  I saw that it was released a
few days ago. That did not solve the problem, but fixed many others, thank
you.

However I found the root cause of the problem: Firefox.
When html is generated by wasm within Firefox, the autofocus attribute
fails to give focus to the element. When the wasm generated html is copied
to a file and served as html, the <input autofocus> correctly gives focus
to the element. I also verified this does not work with a simple test
written in pure web-sys.

In all cases, Chrome correctly gives focus to the <input> element. Leading
me to believe the issue is in Firefox's wasm rendering.
